modules/standard/Time.chpl

```diff
@@ -606,7 +614,8 @@ module Time {
     chpl_day = f.read(int);
   }
 
-  /* Reads this `date` from ISO 8601 format: YYYY-MM-DD */
+  /* Reads this `date` from format corresponding to
```

Comment:
There's a few options I can think of here:
- Just duplicate the format string here again, and say something like "Reads this `date` from input in the format `YYYY-MM-DD`".
- Just change the phrasing but still mention cast to string.
- Leave out the format string here but link this to the corresponding `writeThis`. Since there's only one `writeThis` per type that would work properly.

I think out of those I would prefer the first as it's just immediately clear to a user and doesn't require following a trail. The third is also pretty appealing though, but I think a user can safely assume the format will be compatible with `writeThis` unless for some reason we explicitly state otherwise.

Reply:
I find the third option pretty appealing, especially since as you point out its pretty reasonable to assume that `readThis` and `writeThis` use the same format. However, I don't have strong objections to just doing the first option either.

Reply:
I'd suggest "Reads this `date` with the same format used by :proc:\`date.writeThis\` "

Reply:
Ok, that sounds good to me